src/services/review/RepoCheckerService.ts

```diff
@@ -705,8 +739,16 @@ export default class RepoCheckerService {
 
   mediaPathDecoder = (mediaPath: string) => {
     for (let i = 0; i < ALLOWED_FILE_EXTENSIONS.length; i += 1) {
-      if (mediaPath.toLowerCase().endsWith(ALLOWED_FILE_EXTENSIONS[i])) {
-        return decodeURI(mediaPath)
+      let pathWithoutQueryParams = mediaPath
+      if (mediaPath.includes("?")) {
```

Comment:
you might need to handle url encoded strings here too

Reply:
hey have modified this in this commit [6a6fc29](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/6a6fc296ed62a9292c47c50ad3dc0917908c8d85)

Reply:
can't see the commit but marking as resolved

Reply:
disappeared after rebase [8bb11a1](https://github.com/isomerpages/isomercms-backend/pull/1187/commits/8bb11a12e16b22fac09cfe5aaf1b4e79f919d0b6)